website/react-magma-docs/src/pages/api-intro/page-templates.mdx

```diff
@@ -15,56 +15,129 @@ This template is ideal for applications that need as much of the screen as possi
   <img src="../../images/layout/template-full-width.png" alt=" " />
 </figure>
 
-```noRender
+```tsx
 import React from 'react';
 import styled from '@emotion/styled';
 import { Header } from '@cengage-patterns/header';
-import { magma, useMediaQuery } from 'react-magma-dom';
-
-const Container = styled.div`
-  display: grid;
-  grid-template-columns: auto 1fr;
-  grid-template-rows: auto 1fr;
+import {
+  Drawer,
+  DrawerPosition,
+  Grid,
+  GridAlignContent,
+  GridItem,
+  GridJustifyContent,
+  magma,
+  useMediaQuery,
+} from 'react-magma-dom';
+
+const Container = styled(Grid)`
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: 0.2fr 1.5fr 1.2fr 0.8fr;
   grid-template-areas:
-    'header header'
-    'content content';
-  gap: 2px;
-`;
-
-const StyledHeader = styled(Header)`
-  grid-area: header;
+    'header header header header'
+    'main main main main'
+    'card1 card1 card2 card3'
+    'footer footer footer footer';
+  gap: 0.5em;
```

Comment:
This is okay, but our Grid component supports each of these properties as GridProps. Not sure if we should use that instead? 

```
<Grid
      gridJustifyContent={GridJustifyContent.center}
      gridTemplateAreas={
        ('header header header header'
        'main main main main',
        'card1 card1 card2 card3',
        'footer footer footer footer')
      }
      gridTemplateColumns="repeat(4, 1fr)"
      gridGap="0.5em"
    >
```